locales/gu_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -123,14 +124,14 @@ msgstr ""
 #. at least in:
 #. * https://duckduckgo.com/about. Appears in the window title when you open the link
 msgid "About DuckDuckGo"
-msgstr ""
+msgstr "ડકડકગો વિષે"
```

Comment:
We prefer to have "DuckDuckGo" untranslated. For example, see: https://github.com/duckduckgo/duckduckgo-locales/blob/2dd90431014eb5e77952d401283edd7e1b1fd48e/locales/zh_CN/LC_MESSAGES/duckduckgo.po#L123-L126

Could you please update this and the other places where "DuckDuckGo" is translated?